docker-hub/api/latest.yaml

```
@@ -570,6 +587,43 @@ paths:
           description: Forbidden - this API is only available to users on Pro or Team plans
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /v2/namespaces/{namespace}/repositories/{repository}/images/{digest}/tags:
+    get:
+      tags: [Images]
+      summary: Get image's tags
+      description: Gets current and history tags for an image.
```

```suggestion
      description: Gets current and historical tags for an image
```
for consistency with "Get Details of an image" description